addon/components/inputs/link.js

```diff
@@ -15,10 +16,63 @@ export default AbstractInput.extend({
 
   // == Computed Properties ====================================================
 
-  // We totally don't care about this cause it's view schema
-  @readOnly
+  @computed('cellConfig', 'formValue', 'value')
+  linkLabel (cellConfig, formValue, value) {
+    const rendererLabel = _.get(cellConfig, 'renderer.label')
+
+    if (!rendererLabel) {
+      return value
+    }
+
+    const bunsenId = this.get('bunsenId')
+    const mutableFormValue = formValue ? formValue.asMutable({deep: true}) : {}
+
+    return parseVariables(mutableFormValue, rendererLabel, bunsenId, true)
+  },
+
   @computed('cellConfig', 'value')
-  linkLabel (cellConfig, value) {
-    return _.get(cellConfig, 'renderer.label') || value
+  route (cellConfig) {
+    return _.get(cellConfig, 'renderer.route')
+  },
+
+  @computed('cellConfig', 'formValue', 'value')
+  url (cellConfig, formValue, value) {
+    const rendererUrl = _.get(cellConfig, 'renderer.url')
+
+    if (!rendererUrl) {
+      return value
+    }
+
+    const bunsenId = this.get('bunsenId')
+    const mutableFormValue = formValue ? formValue.asMutable({deep: true}) : {}
+
+    return parseVariables(mutableFormValue, rendererUrl, bunsenId, true)
+  },
+
+  // == Functions ==============================================================
+
+  formValueChanged (newValue) {
+    if (this.get('isDestroyed') || this.get('isDestroying')) {
+      return
+    }
+
+    const cellConfig = this.get('cellConfig')
+    const rendererLabel = _.get(cellConfig, 'renderer.label')
+    const rendererUrl = _.get(cellConfig, 'renderer.url')
+    const labelContainsReferences = rendererLabel && rendererLabel.indexOf('${') !== -1
+    const urlContainsReferences = rendererUrl && rendererUrl.indexOf('${') !== -1
+
+    if (!labelContainsReferences && !urlContainsReferences) {
+      return
+    }
+
+    this.set('formValue', newValue)
```

Comment:
Since `formValue` is used in the above computed property, do you think it'll be worthwhile to diff against the changes to the label and url references. Otherwise, it'll cause a re-render for every form value change but not a big deal unless you expect to see tons of these on a single form.